src/pages/notifications/index.vue

```diff
@@ -83,6 +83,12 @@
           @click.native="handleClickEvent"
           @click.native.stop
         >
+          <client-only>
+            <lazy-component
+              class="absolute inset-0 pointer-events-none -top-full"
```

Comment:
for using `absolute` position, you need a parent (probably is `<LinkV2/>` in this case) with a `relative` position, otherwise the bound of this `<lazy-component/>` will be unpredictable (I guess it is `<body/>`)